### PR TITLE
add extension dockerfile v0.2-1 for sagemaker autopilot container

### PR DESCRIPTION
**Pull request by wiltonwu**
*Description of changes:*
* Adding instructions for new version of `sagemaker-sklearn-automl` container used in SageMaker Autopilot
* Similar to existing code in 0.21-1 branch https://github.com/aws/sagemaker-scikit-learn-container/tree/0.21-1/docker/0.21.2/extension
* Tested by building locally

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: ci/buildspec-extension.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      python: 3.7
+  pre_build:
+    commands:
+    - echo Logging in to Amazon ECR...
+    - $(aws ecr get-login --no-include-email --region $AWS_DEFAULT_REGION)
+  build:
+    commands:
+    - echo Build started on `date`
+    - echo Building the Docker image...
+    - docker build -t sklearn-base:$FRAMEWORK_VERSION-cpu-py3  -f docker/$SKLEARN_FRAMEWORK_VERSION/base/Dockerfile.cpu .
```
Comment (aws-patlin) [inline ci/buildspec-extension.yml:21]: `sklearn-base:$FRAMEWORK_VERSION-cpu-py3` should be `sklearn-base:$SKLEARN_FRAMEWORK_VERSION-cpu-py3`

File: ci/buildspec-extension.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    - pip install wheel setuptools
+    - python setup.py bdist_wheel
+    - docker build -t preprod-sklearn:$FRAMEWORK_VERSION-cpu-py3 -f docker/$SKLEARN_FRAMEWORK_VERSION/final/Dockerfile.cpu .
```
Comment (aws-patlin) [inline ci/buildspec-extension.yml:24]: `preprod-sklearn:$FRAMEWORK_VERSION-cpu-py3` should be `preprod-sklearn:$SKLEARN_FRAMEWORK_VERSION-cpu-py3`

File: docker/0.23-1/extension/Dockerfile.cpu [hunk 3]
```diff
@@ -0,0 +1,9 @@
+FROM preprod-sklearn:0.23-1-cpu-py3
+
+RUN pip freeze | grep -q 'scikit-learn==0.23.2'; \
+	if [ $? -eq 0 ]; \
+		then echo 'scikit-learn version 0.23.2 requirement met'; \
+		else echo 'ERROR: Expected scikit-learn version is 0.23.2, check base images for scikit-learn version' && \
+			 exit 1; fi
+
+RUN pip install --upgrade --no-cache --no-deps sagemaker-scikit-learn-extension==2.0.0
```
Comment (aws-patlin) [inline docker/0.23-1/extension/Dockerfile.cpu:9]: Nit: Missing newline

File: ci/buildspec-extension.yml (hunk 1 above)
Comment (wiltonwu) [reply ci/buildspec-extension.yml]: Thanks for the catch!

**Pull request merged**